### ISSUE TITLE
Clone and Run Server doesn't work.

### ISSUE THREAD

**Issue by csalvato** [rawsyntax/rails-angular-example#1]
Cloning this repo and running `rails server` doesn't work.  Running `rails new .` and pushing `n` to all overwrites will allow server to run but produces an error on http://localhost:3000/example.

```
Showing /Users/csalvato/Development/Web/rails_projects/angular_example2/rails-angular-example/app/views/layouts/application.html.erb where line #6 raised:
couldn't find file 'angular'
  (in /Users/csalvato/Development/Web/rails_projects/angular_example2/rails-angular-example/app/assets/javascripts/application.js:3)
```

**Comment by rawsyntax** [comment]
there's a companion blog post http://rawsyntax.com/blog/how-to-set-up-angular-with-rails/ which details some other commands you have to run first

**Comment by csalvato** [comment]
I found the GitHub repo through that blog post.  Ran it by cloning the repo
and by going through your blog post.  Both produce the same result.

**Comment by rawsyntax** [comment]
@csalvato if you run `bundle exec rake bower:install` like in the blog post and then create the directory structure (under Organize the Angular App) you can then start it and run it with no errors.

**Comment by csalvato** [comment]
Cloned this repo.  Performed `bundle exec rake bower:install`:

```
$ bundle exec rake bower:install
/usr/local/bin/bower install -p
bower not-cached    git://github.com/angular/bower-angular.git#v1.2.25
bower resolve       git://github.com/angular/bower-angular.git#v1.2.25
bower download      https://github.com/angular/bower-angular/archive/v1.2.25.tar.gz
bower extract       angular#v1.2.25 archive.tar.gz
bower invalid-meta  angular is missing "ignore" entry in bower.json
bower resolved      git://github.com/angular/bower-angular.git#1.2.25
bower install       angular#1.2.25

angular#1.2.25 bower_components/angular
```

Result:

![Image of Error](http://i.imgur.com/YYQnsoM.png)

**Comment by rawsyntax** [comment]
did you also create the directories?

```
templates/
modules/
filters/
directives/
models/
services/
controllers/
```

**Comment by csalvato** [comment]
Thank you for your patience in dealing with me!

I just checked the original files I created when walking through the blog post and realized that I made the directories, but had them in the wrong folder.  So when that didn't work, I cloned your github repo expecting them to be set up in the repo and that it should just _work_.

Anyway, I put the directories back, and its working now.  Thanks so much!!!
